styles.css

```diff
@@ -0,0 +1,118 @@
+* {
+  font-family: 'Roboto', Helvetica, sans-serif;
+}
+
+body {
+  background-color: #191765;
+  max-width: 375px;
+  background-image: url(images/Ellipse\ 18.svg);
+  background-repeat: no-repeat;
+  background-position: 0 47px;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+#header_list {
+  display: flex;
+  justify-content: space-between;
+  color: #fff;
+  align-items: center;
+}
+
+#company {
+  font-size: 18px;
+}
+
+#presentation {
+  display: flex;
+  margin-left: 24px;
+  margin-top: 60px;
+  margin-right: 24px;
+}
+
+#pres {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  max-width: 327px;
+  text-align: center;
+}
+
+#name {
+  font-size: 48px;
+  color: #7d73fc;
+  font-weight: 700;
+  margin-bottom: 12px;
+}
+
+#position {
+  font-size: 32px;
+  font-weight: 500;
+  color: #edecff;
+  margin-bottom: 40px;
+}
+
+#descr {
+  font-size: 20px;
+  font-weight: 400;
+  color: #c8c8d4;
+}
+
+#qualif_group {
+  display: flex;
+  flex-wrap: wrap;
+  background-image: url(images/Ellipse\ 19.svg);
+  background-repeat: no-repeat;
+  background-position: 290px 0;
+}
+
+.qualif {
+  display: flex;
+  flex-direction: column;
+  width: 35%;
+  margin-left: 44px;
+}
+
+.qualif_title {
+  font-size: 12px;
+  font-weight: 500;
+  color: #9490ff;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 24.5px;
+}
+
+.qualif_item {
+  font-size: 15px;
+  font-weight: 400;
+  color: #f5f5ff;
+  margin-bottom: 19.11px;
+}
+
+#links {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 98.51px;
+  margin-right: 97.5px;
+}
```

Comment:
Your social media icons are to space out and they are learn too much on the right please stick to the design in the figma template you picked. 
your design:
![Screenshot from 2022-04-21 10-44-46](https://user-images.githubusercontent.com/79506105/164390907-0592b7c8-3904-4379-9368-2412fc8e8793.png)
But dont worry Im here to help you. Kindly try this to help you align your social media icons properly.
#links {
  display: flex
  margin-left : 0;
  margin-right: 20px
 margin-top: 20px
gap: 20px
padding-left: 118px;
}
Try that out thanks